src/Invoices/InvoicesContent.php

```diff
@@ -91,8 +91,8 @@ final class InvoicesContent extends DateAwareEntity
     private $brutto;
 
     /**
-     * @var string
-     * @JMS\Type("string")
+     * @var int
+     * @JMS\Type("integer")
```

Comment:
Czy na pewno integer. Dokumentacja twierdzi, że string (jako przykład podano WDT):

```
Stawka VAT - pole dopuszczalne tylko w przypadku polskich stawek VAT (23, WDT itp). W przypadku stawek MOSS należy obligatoryjnie stosować strukturę <vat_code></id>Tutaj id stawki</id></vat_code>. Lista stawek dostępna jest w akcji /vat_codes/find.
```

Reply:
@jacekkow @mgralikowski dzięki za pomoc. Dodałem `VatPlRate`, żeby łatwo operować na podstawowych polskich stawkach (dodałem ZW oraz WDT) oraz dodałem możliwość tworzenia obiektu `Price` z `VatCodeId`. Sprawdźcie proszę, czy tak to może wyglądać.